package.json

```diff
@@ -3,5 +3,21 @@
   "private": "true",
   "scripts": {
     "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "webpack": "^5.89.0",
+    "webpack-cli": "^5.1.4"
+  },
```

Comment:
- [ ] There are couple of errors when I run your `App` component (see image) all related to some packages missing (`axios`, `redux`, etc). Kindly make sure all the packages that you use are present under the `dependencies` module in the `package.json` file
![Screenshot from 2023-11-28 12-19-36](https://github.com/BranBayou/hello-rails-react/assets/91142432/d83764b1-912e-4d3f-85c3-084fa8196233)